### PR TITLE
Fixed increments for subsequent services.

### PR DESCRIPTION
**Pull request by huwshimi**
Fix increments for the subsequent (different) services.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- add a service e.g. mysql
- add it again
- you should have two services: mysql and mysql2
- add a different service e.g. wordpress
- add it again
- you should get two more services: wordpress and wordpress2

**Comment by hatched** [comment]
:+1: QA :ok:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3512/
Test PASSed.

File: jujugui/static/gui/src/app/models/models.js [hunk 1]
```diff
@@ -762,7 +758,13 @@ YUI.add('juju-models', function(Y) {
           // The first service that is created won't have a count.
           return count || 1;
         }
+        return 0;
       }));
+      // The first service for a charm shouldn't get a count, but subsequent
+      // services should.
+      if (highestId < 1 || isNaN(highestId) || highestId === null) {
```
Comment (fabricematrat) [inline jujugui/static/gui/src/app/models/models.js]: I don't think it can be null but this doesn't really matter.

**Comment by fabricematrat** [comment]
:+1:  thanks.

**Comment by jaycee** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1205

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1207

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**